docs/resources/Channel.md

```diff
@@ -531,6 +552,19 @@ There are multiple message types that have a message_reference object.  Since me
 - These are automatic messages sent when a message is pinned (type 6).
 - These messages have `message_id` and `channel_id`, and `guild_id` if it is in a guild, which point to the message that was pinned.
 
+###### Forwards
+
+- These are messages which capture a snapshot of a message.
+- These messages have an array of [`message_snapshots`](#DOCS_RESOURCES_CHANNEL/message-snapshot-object) field containing a copy of the original message. This copy follows the same structure as a message, but has only the minimal set of fields returned required for context/rendering.
+  - of note: `author` will be excluded
```

Comment:
```suggestion
- These messages have an array of [`message_snapshots`](#DOCS_RESOURCES_CHANNEL/message-snapshot-object) field containing a copy of the original message. This copy follows the same structure as a message, but only has the minimum set of fields returned required for context/rendering (notably, the `author` field is excluded)
```

Reply:
I feel like this bullet point sticks out more, which is the intent